<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRooms</class>
 <widget class="QDialog" name="EditRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>726</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>952</width>
    <height>726</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>952</width>
    <height>726</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#editWarnLabel{
	color: red;
}</string>
  </property>
  <widget class="QTableWidget" name="editRoom">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>99</y>
     <width>930</width>
     <height>192</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>930</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>921</width>
     <height>1000</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="editWarnLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>680</y>
     <width>931</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>380</y>
     <width>351</width>
     <height>289</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="curStart">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Checkout Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="curEnd">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="curProgram">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Room</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="curRoom">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current  Cost</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="curCost">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_13">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adjusted Cost</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="curAdjust">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_16">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Final Cost</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="curFinal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>351</width>
     <height>289</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="swapStart">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Checkout Date</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="swapEnd">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="swapProgram">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Room</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="swapRoom">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_15">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current  Cost</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="swapCost">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Adjusted Cost</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="swapAdjust">
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_17">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Final Cost</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="swapFinal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>148</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>125</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="editSwap">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>55</y>
     <width>250</width>
     <height>37</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>35</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Switch to Swap Mode</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>931</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_18">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current Task:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="editMoveType">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>55</y>
     <width>216</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Program</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="editProgram">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>294</y>
     <width>342</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current Client</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="currentUse">
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>294</y>
     <width>342</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Swapped Client</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="swapUser">
      <property name="minimumSize">
       <size>
        <width>340</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>630</y>
     <width>229</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="editOkButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="editCancelButton">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>35</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
